packages/components/search/README.md

```diff
@@ -0,0 +1,131 @@
+<h1 align="center">
```

Comment:
Can you please add `properties`, `slots` and `events` to the readme file like https://github.com/Tradeshift/elements/blob/master/packages/components/aside/README.md